utilities/generate_cluster_configs_from_vagrant_hosts.py

```diff
@@ -10,26 +10,22 @@
 from libraries.utilities.generate_clusters_from_pool import generate_clusters_from_pool
 
 
-def generate_cluster_configs_from_vagrant(private_network, public_network):
+def generate_cluster_configs_from_vagrant(private_network, public_network, public_network_ethernet):
     """
     1. Gets the status for a running vagrant vm set.
     2. Uses the host name to look up the ip allocated to each vagrant vm instance
     3. Uses this IP list to build a pool.json file and generate the cluster configurations
     """
 
-    if private_network and public_network:
-        raise ProvisioningError("Invalid private_network and public_network flags")
-
-    if not private_network and not public_network:
-        raise ProvisioningError("Invalid private_network and public_network flags")
-
```

Comment:
Why did you move this from the method into main? If someone was importing / calling this from another script, this validation would not longer be run.